msteams-platform/toolkit/TeamsFx-SDK.md

```diff
@@ -188,183 +234,74 @@ try {
-const credential = teamsfx.getCredential(); //Create an API Client that uses SSO token to authenticate requests
-const apiClient = CreateApiClient(teamsfx.getConfig("apiEndpoint")),
-new BearerTokenAuthProvider(async () =>  (await credential.getToken(""))!.token);// Call API hosted in Azure Functions on behalf of user
-const response = await apiClient.get("/api/" + functionName);
-```
-
-</details>
-
-<br>
-
-<details>
-<summary><b>Access SQL database in Azure Function</b></summary>
-
-Use `tedious` library to access SQL and leverage `DefaultTediousConnectionConfiguration` that manages authentication.
-Apart from `tedious`, you can also compose connection config of other SQL libraries based on the result of `sqlConnectionConfig.getConfig()`.
-
-```ts
-// Equivalent to:
-// const sqlConnectConfig = new DefaultTediousConnectionConfiguration({
-//    sqlServerEndpoint: process.env.SQL_ENDPOINT,
-//    sqlUsername: process.env.SQL_USER_NAME,
-//    sqlPassword: process.env.SQL_PASSWORD,
-// });
-const teamsfx = new TeamsFx();
-// If there's only one SQL database
-const config = await getTediousConnectionConfig(teamsfx);
-// If there are multiple SQL databases
-const config2 = await getTediousConnectionConfig(teamsfx "your database name");
-const connection = new Connection(config);
-connection.on("connect", (error) => {
-  if (error) {
-    console.log(error);
-  }
-});
-```
-
-</details>
-
-<br>
-
-<details>
-<summary><b>Use certificate-based authentication in Azure Function</b></summary>
-
-```ts
-const authConfig = {
-  clientId: process.env.M365_CLIENT_ID,
-  certificateContent: "The content of a PEM-encoded public/private key certificate",
-  authorityHost: process.env.M365_AUTHORITY_HOST,
-  tenantId: process.env.M365_TENANT_ID,
-};
-const teamsfx = new TeamsFx(IdentityType.App);
-teamsfx.setCustomeConfig({
-  certificateContent: "The content of a PEM-encoded public/private key certificate"
-});
-const token = teamsfx.getCredential().getToken();
-```
-
-</details>
-
-<br>
-
-<details>
-<summary><b>Use Graph API in bot application</b></summary>
+    ```ts
+    import {
+      createMicrosoftGraphClient,
+      TeamsFx,
+    } from "@microsoft/teamsfx";
+    ```
 
-Add `TeamsBotSsoPrompt` to dialog set.
+    2. Use `TeamsFx.login()` to get user consent.
 
-```ts
-const { ConversationState, MemoryStorage } = require("botbuilder");
-const { DialogSet, WaterfallDialog } = require("botbuilder-dialogs");
-const { TeamsBotSsoPrompt } = require("@microsoft/teamsfx");
+    ```ts
+    // Put these code in a call-to-action callback function to avoid browser blocking automatically showing up pop-ups.
+    await teamsfx.login(["User.Read"]); // Login with scope
+    ```
 
-const convoState = new ConversationState(new MemoryStorage());
-const dialogState = convoState.createProperty("dialogState");
-const dialogs = new DialogSet(dialogState);
+    3. Initial a TeamsFx instance and graph client and get information from MS Graph by this client.
```

Comment:
Should be `Initialize a TeamsFx instance..`?

Reply:
Yes, it should be the verb. here.

Reply:
Updated comment. It shows Initialize.